sandbox/OpenIddict.Sandbox.AspNetCore.Client/Controllers/AuthenticationController.cs

```diff
@@ -20,7 +20,8 @@ public ActionResult LogIn(string provider, string returnUrl)
             !string.Equals(provider, Providers.GitHub,  StringComparison.Ordinal) &&
             !string.Equals(provider, Providers.Google,  StringComparison.Ordinal) &&
             !string.Equals(provider, Providers.Reddit,  StringComparison.Ordinal) &&
-            !string.Equals(provider, Providers.Twitter, StringComparison.Ordinal))
+            !string.Equals(provider, Providers.Twitter, StringComparison.Ordinal) &&
```

Comment:
Please remove any change to the sample as the Cognito provider is not going to be enabled by default.